src/app/view/variant/variant-view/variant-history/version-entry-layout/version-entry-layout.component.scss

```diff
@@ -70,3 +70,22 @@
     border: none;
   }
 }
+
+
+.user-id {
+  width: 140px;             /* Set the desired width */
+  overflow: auto;         /* Hide overflowing content */
+  text-overflow: ellipsis;  /* Display ellipsis (...) for overflow */
+  white-space: nowrap;      /* Prevent text from wrapping */
+  user-select: text;        /* Allow text selection */
+}
+
+.user-id {
+  -ms-overflow-style: none; /* for Internet Explorer, Edge */
+  scrollbar-width: none; /* for Firefox */
+  overflow-y: scroll;
```

Comment:
Hm, what is the purpose of these rules to enable the scrollbar and then hide it?